apiextensions/storageversion/migrator_test.go

```diff
@@ -63,6 +63,21 @@ var (
 			},
 		},
 	}
+
+	singleVersionFakeCRD = &apix.CustomResourceDefinition{
```

Comment:
nit: just pull this into `TestMigrate_SingleStoredVersion` if it's only used there